schematic/store/synapse.py

```diff
@@ -100,14 +101,32 @@ def __init__(
 
         self._query_fileview()
 
+    def _purge_synapse_cache(self):
+        # try clearing the cache
+        # scan a directory and check size of files
+        if os.path.exists("/var/www/.synapseCache"):
```

Comment:
There is a way of getting the synapseCache dir using the synapse connection object.  If that exists, maybe use that.

Reply:
Thanks Tom. I think the function is: `self.syn.cache.cache_root_dir`

Reply:
@thomasyu888 hmm if we change this line to `self.syn.cache.cache_root_dir`, I think this will impact local cache when running the endpoint locally? Should we avoid that by using `/var/www/.synapseCache` instead?

Reply:
^ That is up to you.  It might make most sense to hard code it if you only expect the cache dir to be in `/var/www/.synapseCache`